opm/autodiff/WellInterface.hpp

```diff
@@ -82,7 +83,13 @@ namespace Opm
         typedef double Scalar;
 
         typedef Dune::FieldVector<Scalar, numEq    > VectorBlockType;
+#if  DUNE_VERSION_NEWER_REV(DUNE_ISTL, 2 , 5, 1)
+        // 3x3 matrix block inversion was unstable from at least 2.3 until and
+        // including 2.5.0
+        typedef Dune::FieldMatrix<Scalar, numEq, numEq > MatrixBlockType;
+#else
         typedef Dune::FieldMatrix<Scalar, numEq, numEq > MatrixBlockType;
```

Comment:
This line made me wonder: I thought we used the MatrixBlock class by default here, for the precise reason stated in the comment? However it seems to have been removed at some point, both here (blocks for the well equations) and in BlackoilModelEbos (blocks for the reservoir equations).

Are we sure that the 3x3 and 4x4 inversions are good enough?

As written now, this part of the PR is of course superfluous.

Reply:
Gone in favor of using Dune::FMatrixHelper::invertMatrix.